app/services/obituary_service.rb

```diff
@@ -21,15 +21,20 @@ def get_recent_obituaries
     JSON.parse(resp.body, symbolize_names: true)
   end
 
-  def get_advanced_obituaries(name, year)
+  def advanced_search_name_and_year(name, year)
```

Comment:
changed the name since there was another advanced search method created just for year.